### PR TITLE
LOC CHECKIN | Microsoft/visualfsharp master | 20180409

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/fsharp/xlf/FSComp.txt.de.xlf [hunk 1]
```diff
@@ -6984,7 +6984,7 @@
       </trans-unit>
       <trans-unit id="ilreadFileChanged">
         <source>The file '{0}' changed on disk unexpectedly, please reload.</source>
-        <target state="new">The file '{0}' changed on disk unexpectedly, please reload.</target>
+        <target state="translated">Die Datei "{0}" wurde auf dem Datenträger unerwartet geändert. Laden Sie sie erneut.</target>
```
Comment (realvictorprm) [inline src/fsharp/xlf/FSComp.txt.de.xlf]: I would suggest to change the last bit to "Bitte laden Sie _diese_ erneut."

File: src/fsharp/xlf/FSStrings.de.xlf [hunk 2]
```diff
@@ -1424,7 +1424,7 @@
       </trans-unit>
       <trans-unit id="EnumMatchIncomplete1">
         <source>Enums may take values outside known cases.</source>
-        <target state="new">Enums may take values outside known cases.</target>
+        <target state="translated">Enumerationen können Werte außerhalb bekannter Fälle verwenden.</target>
```
Comment (realvictorprm) [inline src/fsharp/xlf/FSStrings.de.xlf]: Here I would rather suggest "Enumerationen können Werte außerhalb _definierter_ Fälle _annehmen_."

**Comment by brettfo** [comment]
@realvictorprm: We're still working through what the process looks like for community-supplied translations, but to keep our tooling working I need to merge this as-is.  Can you create a PR against master with your suggestions?  I can use it as ammo to get a workable process put in place.

**Comment by realvictorprm** [comment]
OK thanks for letting me know @brettfo, will do.

**Pull request merged**